sws.go

```diff
@@ -68,6 +73,12 @@ func main() {
 
 }
 
+func startServer() {
+	log.Info("SWS: Starting http server")
+	router := routing.NewRouter()
+	glog.Fatal(http.ListenAndServe(":8080", router))
```

Comment:
So `glog.Fatal` will be called even if there's no error?

Reply:
ListenAndServe blocks and always returns an err.

This is going to have to change so we get this and filesrever on the same host/port. I'm going to look into this when I refactor to get "fileserver" configured to be "server" so we have the same binding host/port.

Reply:
@xeviknal once you rebase on master to pick up the PR #8 I can review